plexapi/utils.py

```diff
@@ -323,7 +323,7 @@ def toDatetime(value, format=None):
                 return None
         else:
             try:
-                return datetime.utcfromtimestamp(0) + timedelta(seconds=int(value))
+                return datetime.fromtimestamp(0) + timedelta(seconds=int(value))
```

Comment:
Here you calculate timezone offset from a winter date (1970-01-01) and add timedelta.
Therefore this will return a wrong time for a summer `value`.

Reply:
The only thing I can think of is to use `datetime.fromtimestamp(value)` and catch `OSError` or `OverflowError` to fallback to `datetime.fromtimestamp(0) + timedelta(seconds=int(value)` if the `value` is negative or very large.

Reply:
I don't what was wrong with the previous code :
```python
value = min(max(int(value), 86400), 2**31 - 1)
value = datetime.fromtimestamp(int(value))
```
We seem to agree with the `datetime.fromtimestamp(value)` part.
Remains the handling of offlimit values.

Reply:
Right, the same as previous code, but instead of the hacky max/min, just catch the exception and use the `timedelta` from 0.

For normal values it would take into account the timezone, but out-of-bounds values won't use the correct timezone. If the values are out-of-bounds then something else has probably gone wrong and a small timezone error is probably the least of your worries.

Reply:
> Right, the same as previous code, but instead of the hacky max/min, just catch the exception and use the `timedelta` from 0.

Should I change anything else? AFAICT this is what's implemented atm with `except OverflowError`.